assets/styles/styles.css

```diff
@@ -340,15 +336,15 @@ strong {
     overflow-x: hidden;
   }
 
-  ._last-middle {
+  .last-middle {
     margin: 0 20px;
   }
 
   .last-sect-start {
     width: 100%;
   }
 
-  ._contries {
+  .contries {
     width: 100%;
   }
 }
```

Comment:
Try to use significant class names in your code, please. And also try not to make your code bulky